alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/MessageReceiver.java

```diff
@@ -39,6 +40,7 @@ public void onMessage(Message message) {
                 T event = gson.fromJson(textMessage.getText(), eventClass);
                 logger.trace("{} event {}", receiverClassName, event);
                 handleEvent(event);
+                logger.debug("Received Event ID: {}", event.getEventId());
```

Comment:
I think it might make more sense to log this before handling the event. If a `RuntimeException` is thrown, we'll still know the event ID at _debug_.

Reply:
Good idea